charts/cluster-api-provider-openstack/templates/validatingwebhookconfiguration-0.yaml

```diff
@@ -13,15 +13,15 @@ webhooks:
     service:
       name: capo-webhook-service
       namespace: capo-system
-      path: /validate-infrastructure-cluster-x-k8s-io-v1alpha7-openstackcluster
+      path: /validate-infrastructure-cluster-x-k8s-io-v1beta1-openstackcluster
```

Comment:
Note to self: A->B sounds fraught with peril, nvesigate

Reply:
Okay, this doesn't make sense, this does not exist in the '0.9.0' tag (and neither do the v1beta1 CRDs, which is my concern), but does on 'main'.  Someone (not you) has fucked something up!

Reply:
Confusing myself evidently 😸  Mondays!